### PR TITLE
fix: link preview with image displayed horizontal rather than vertical

### PR DESCRIPTION
**Pull request by Bilb**
https://user-images.githubusercontent.com/1544279/187347333-92a9a072-3cb2-463b-b500-37dc7ade9497.mp4

### CODE REVIEW

File: stylesheets/_modules.scss [hunk 1]
```diff
@@ -156,6 +161,7 @@
   flex-direction: row;
   align-items: flex-start;
   border: 1px solid $color-black-015;
+  flex-grow: 1;
```
Comment (yougotwill) [inline stylesheets/_modules.scss:164]: Just me preference but I like to group CSS properties together. So flex-grow could be after align-items in this case.

File: ts/components/conversation/message/message-content/MessageContent.tsx [hunk 2]
```diff
@@ -230,33 +228,17 @@ export const MessageContent = (props: Props) => {
...
-
-    if (isImageAttachment(first.image) && width && width >= MINIMUM_LINK_PREVIEW_IMAGE_WIDTH) {
-      const dimensions = getImageDimensionsInAttachment(first.image);
-      if (dimensions) {
-        return dimensions.width;
-      }
-    }
-  }
+  // we do not care about the presence of a preview or not to hardcode the width.
+  // the width with a preview will be determine by a flexbox
```
Comment (yougotwill) [inline ts/components/conversation/message/message-content/MessageContent.tsx:241]: *will be determined by

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/conversation/message/message-content/MessageContent.tsx (hunk 2 above)
Comment (Bilb) [reply ts/components/conversation/message/message-content/MessageContent.tsx:241]: :+1:

File: stylesheets/_modules.scss (hunk 1 above)
Comment (Bilb) [reply stylesheets/_modules.scss:164]: :+1:

**Review by yougotwill — APPROVED** [review]
LGTM 🚀

**Review by yougotwill — APPROVED** [review]
LGTM

**Comment by Bilb** [comment]
merged into https://github.com/oxen-io/session-desktop/pull/2459

**Pull request closed without merging**